### ISSUE TITLE
Inconsistency between the reports

### ISSUE THREAD

**Issue by JB91451** [Nesvilab/philosopher#117]
Dear Felipe,

I just noticed some inconsistent behaviour between the different report tsv tables when I wanted to investigate my decoys in more detail:
Primarily I noticed, that my decoys in the proteins table can have a certain number of (unique) peptides but never have (unique) spectral counts. To not report spectral counts makes some sense when one wants to calculate NSAF values etc. so I did not mind. However, when I wanted to count the spectra manually in the psm's table or the peptides table, there were no entries for the reported decoy proteins. Instead some other decoy hits were reported, which do not appear in the protein table.
Moreover, in the pep.xml and prot.xml files decoy hits from both tables are present, so I guess this is not a mapping error.
I used the picked-FDR algorithm with peptide and protein level filter and choose to report decoys in philosopher 3.2.3

Best,
Juergen

**Comment by prvst** [comment]
Hi Juergen;

Can you share with me your tables and some specific examples ?

**Comment by JB91451** [comment]
Yes of course. Please find the peptide and protein tables below.

The protein group 1275(a) (protein: Reverse_sp|Seq_LysC_003436|Seq_LysC_003436) would be an example of a decoy protein that occures only in the protein table but not in the peptide list. It is reported with 1 unique and 2 total peptdes but no PSMs

And the protein Reverse_sp|Seq_LysC_005932|Seq_LysC_005932 would be an example that is present in the peptides table but not in the protein table.


[protein.txt](https://github.com/Nesvilab/philosopher/files/4369708/protein.txt)
[peptide.txt](https://github.com/Nesvilab/philosopher/files/4369709/peptide.txt)

**Comment by prvst** [comment]
please allow me some time to investigate this issue.

**Comment by JB91451** [comment]
Hi Felipe,
I hope you are doing well. Did you figure out anything about that issue in the meantime? Please let me know if you need any additional files or information.
Best,
Juergen

**Comment by prvst** [comment]
Hi Juergen;

Sorry but not yet, I'm dedicating some time to replying our reviewers. I'll keep you posted.

**Comment by JB91451** [comment]
Nice to hear that your publication makes some progress (is it the one about the Philosopher toolkit?). Perhaps I can change the citation from this webpage to your paper in our current manuscript when we get the reviewers comments.

**Comment by prvst** [comment]
Yes, we prepared a publication for the Philosopher toolkit. I appreciate the effort in wanting to use the citation to the paper, but it might still take a couple of weeks until we have the acceptance. I'll let you know.

**Comment by prvst** [comment]
Please, try the latest release. In case you find issues again, feel free to reopen this.

**Comment by JB91451** [comment]
Dear Felipe,

I tried with the version 3-3-10 and the same pepXML/protXML files but the error persists.

The commands I executed were:

/philosopher_3310/philosopher workspace --clean
/philosopher_3310/philosopher workspace --init
/philosopher_3310/philosopher database --annotate 180921_iPtgxDB_LysC_Merge_ArgC_shortHeader_Cont_fw_rev_PhilosopherHeader.fasta --prefix Reverse_ --enzyme trypsin
/philosopher_3310/philosopher filter --pep 0.01 --pepxml interact.pep.xml --picked --prot 0.01 --protxml interact.prot.xml --psm 0.01 --tag Reverse_
/philosopher_3310/philosopher report --decoys

Unfortunately, the XML files and the database are to large to upload them here. 

Best,
Juergen

P.S.: It seems that I can not reopen a topic if someone else closed it.
